src/test/script_tests.cpp

```diff
@@ -955,7 +970,14 @@ BOOST_AUTO_TEST_CASE(script_json_test)
         std::string scriptSigString = test[pos++].get_str();
         CScript scriptSig = ParseScript(scriptSigString);
         std::string scriptPubKeyString = test[pos++].get_str();
-        CScript scriptPubKey = ParseScript(scriptPubKeyString);
+        CScript scriptPubKey;
+        // If requested, auto-generate the taproot output
+        if (strcmp(scriptPubKeyString.c_str(), "0x51 0x20 <AUTOGEN:TAPROOTOUTPUT>")== 0) {
```

Comment:
Would it makes sense to just search for the string `<AUTOGEN:TAPROOTOUTPUT>`? And omit `0x51 0x20`

Reply:
That works as well and I had originally wrote it that way. My concern was that is might be confusing and since all TAPROOT outputs start as "0x51 0x20" this would be less confusing.